src/org/oidc/common/ValueError.java

```diff
@@ -0,0 +1,7 @@
+package org.oidc.common;
+
+public class ValueError extends Exception{
```

Comment:
It is extending Exception, it shouldn't be called ValueError. In Java, exception and error are very different. An Error is a subclass of Throwable that indicates serious problems that a reasonable application should not try to catch. Most such errors are abnormal conditions.

Please rename it appropriately - so that name ends with Exception. There is no Javadoc here, so I don't know what is the purpose of this custom exception. Could rename this file and add Javadoc?